### DISCUSSION TITLE
All that's left of the spool to get a functional box

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#50] [Show and tell]
![20221028_134515](https://user-images.githubusercontent.com/111667850/198700262-224ffdd8-5c42-41cd-84e5-1bc47835722d.jpg)

That's it. I took a gamble and made the display mount. All the core and octopus tray is printed and and ready to go. Not sure if I'm going to finish the rest of it with another spool. I have a roll of overture "Starry Night" that I wasn't sure about what I'd do with but I think I know now that I'm going to try the monolithic prints with it on the e5+

**Comment by jon-harper** [comment]
I *love* when it comes in under the wire like that. I'm enjoying the pics of someone making another OmniBox, too!
